.github/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] Kindly add linters to your project. In the root project, you need to have a folder named `.github`. Inside this one, you need another folder named `workflows`. And inside the workflows folder, you need a file named `linters.yml`. Inside this file, you need to have the linters. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css);
Here is a video demo I created demonstrating  the above:
>
https://user-images.githubusercontent.com/52934128/164221056-46f41b39-cb1e-4fd9-a7f9-dd47bf835439.mp4
- [ ] Kindly ensure your linter results pass * AKA: Linter should report NO errors.